packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -119,14 +168,14 @@ class Statement extends React.Component {
             map[item.col_index] = item;
             return map;
         }, {});
+        const is_mx_mlt =
+            this.props.landing_company_shortcode === 'iom' || this.props.landing_company_shortcode === 'malta';
 
         return (
             <React.Fragment>
                 <ReportsMeta
-                    i18n_heading={localize('Statement')}
-                    i18n_message={localize(
-                        'View all transactions on your account, including trades, deposits, and withdrawals.'
-                    )}
```

Comment:
Question/just for my understanding:

So does the `Statement` page no longer have a heading and description message even for non-MX/MLT?

Reply:
Completely true, design was changed. That extra description besidethe datepicker component was removed.